docs/ARCHITECTURE.md

```diff
@@ -74,7 +74,11 @@ flowchart LR;
     import --> backend
 ```
 
-The delivered formats are loaded into objects. In general, we expect 2-dimensional data.
+**1st step - Extracting** Delivered formats are loaded into objects. In general we expect 2 dimensional data - the data set.
+
+**2nd step - Loading** A dataset contains multiple tupel. A tupel in turn consists of multiple attributes. Imports my support separate headlines. In case of csv the first row could be a headline. Usually each csv-row is a tupel/data point.  
+
+**3rd step - Mapping** The internal 2-dimensional data grid is mapped with help of associative attribut naming. Further rules, checks and specialised functions are added - see 'offer_mapping'. This allows to handle customer specific representation of boolean values or time string formats.
 
```

Comment:
What about the steps below? I assume you want to delete them?

Reply:
Hi indeed this is a merge-conflict - yes please remove the "First step"/"Second step".